broker/src/main/java/org/apache/rocketmq/broker/client/net/Broker2Client.java

```diff
@@ -62,34 +60,22 @@ public Broker2Client(BrokerController brokerController) {
     }
 
     public void checkProducerTransactionState(
+        final String group,
         final Channel channel,
         final CheckTransactionStateRequestHeader requestHeader,
-        final SelectMappedBufferResult selectMappedBufferResult) {
+        final MessageExt messageExt) throws Exception {
```

Comment:
Can this Exception be specific?

Reply:
Thanks for your review，in fact，actually I don't want to throw any exceptions here，but considering that the encode method may cause exception，so in this method just throws exception following with the encode method.